tests/unit/models/framework/test_entity.py

```diff
@@ -112,6 +112,9 @@ class Response:
     wg = future_wrap(g)
     e = entity(class_)
 
+    orig_parse_reponse = e._parse_response
+    e._parse_response = classmethod(lambda cls, x: (orig_parse_reponse(x) if hasattr(x, "buffer") else x))
```

Comment:
I though tests did fail when I tried to return x only - but doing the test again - indeed this can be removed.